react/features/stream-effects/presenter/JitsiStreamPresenterEffect.js

```diff
@@ -153,6 +153,7 @@ export default class JitsiStreamPresenterEffect {
         this._videoFrameTimerWorker.postMessage({
             id: CLEAR_INTERVAL
         });
+        this._videoFrameTimerWorker.terminate();
```

Comment:
The worker is created only once when the effect is constructed. Worker creation needs to be moved to startEffect.

Reply:
> The worker is created only once when the effect is constructed. Worker creation needs to be moved to startEffect.


I have updated my code.

Reply:
Thank you, can you please sign the CLA so that we can accept the PR.